src/components/Comment/index.tsx

```diff
@@ -305,10 +301,10 @@ export default class CommentComponent extends React.Component<CommentProps> {
                         onClick={onSave}
```

Comment:
I think we should use `<form>` `onSubmit` for the main button submit rather than `onClick`.

Reply:
Thanks, fixed now!